src/pages/Validator/DetailCard.tsx

```diff
@@ -7,38 +7,32 @@ import {HashType} from "../../components/HashButton";
 import RewardsPerformanceTooltip from "../Validators/Components/RewardsPerformanceTooltip";
 import LastEpochPerformanceTooltip from "../Validators/Components/LastEpochPerformanceTooltip";
 import {Types} from "aptos";
-import {
-  useGetMainnetValidators,
-  useGetValidatorToOperator,
-} from "../../api/hooks/useGetValidatorSet";
 import {useGetAccountResource} from "../../api/hooks/useGetAccountResource";
 import {useEffect, useState} from "react";
 import TimestampValue from "../../components/IndividualPageContent/ContentValue/TimestampValue";
+import {useGetMainnetValidators} from "../../api/hooks/useGetMainnetValidators";
 
 type ValidatorDetailProps = {
   address: Types.Address;
 };
 
 export default function ValidatorDetailCard({address}: ValidatorDetailProps) {
-  // make sure that addresses will always start with "0x"
   const addressHex = address.startsWith("0x") ? address : "0x" + address;
```

Comment:
maayan left a [comment](https://github.com/aptos-labs/explorer/pull/307#discussion_r1072971727) on one of my PRs saying that we could use sdk for converting address to hex string. we can do the same thing here.